src/pkg/mcp/utils.go

```diff
@@ -0,0 +1,79 @@
+package mcp
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+
+	"github.com/DefangLabs/defang/src/pkg/mcp/logger"
+)
+
+const (
+	AskDefangBaseURL      = "https://ask.defang.io"
+	DocumentationEndpoint = "data"
+	KnowledgeBaseDir      = "./knowledge_base"
```

Comment:
This is probably not appropriate. We don't want to be writing files to the current working directory. Let's instead write files to `client.userStateDir()`.